src/assets.ts

```diff
@@ -377,22 +391,25 @@ export function createFallbackLaunchConfiguration(): vscode.DebugConfiguration {
         "WARNING17": "*********************************************************************************",
         "preLaunchTask": "build",
         "program": "${workspaceFolder}/bin/Debug/<insert-target-framework-here>/<insert-project-name-here>.dll",
-        "args": [],
+        "args": Array(0),
         "cwd": "${workspaceFolder}",
         "console": "internalConsole",
         "stopAtEntry": false
     };
+
+    return indentJsonString(JSON.stringify(o));
 }
 
 // AttachConfiguration
 export function createAttachConfiguration(): string {
-    return `
-{
-    "name": ".NET Core Attach",
-    "type": "coreclr",
-    "request": "attach",
-    "processId": "\${command:pickProcess}"
-}`;
+    const o = {
```

Comment:
Maybe rename these `o` variables to `configuration`.

Reply:
done